creds/vault/manager.go

```diff
@@ -83,5 +86,14 @@ func (manager VaultManager) NewVariablesFactory(logger lager.Logger) (creds.Vari
 		return nil, err
 	}
 
-	return NewVaultFactory(logger, client, manager.Auth, manager.PathPrefix), nil
+	c := NewAPIClient(logger, client, manager.Auth)
+
+	// TODO: Configurable?
```

Comment:
Yeah, might as well make these configurable. They'd be right at home next to the auth max duration flag from #256.

Also, why 64 seconds? :P